ntbs-service/Pages/LegacyNotifications/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page "/LegacyNotifications/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.LegacyNotifications.Index
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
+                <div class="@notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+    </nhs-grid-row>
+    
+    @if (Model.LegacyImportResult == null)
+    {
+        <div class="ondemand-migration-description-container">
+            <p>You must import this notification to view the full details of the record.</p>
+            <br/>
+            <p>Import may take a few moments.</p>
+            <br/>
+            <p>Do you wish to proceed?</p>
+            <br/>
+            <form method="post">
+                <button class="nhsuk-button">Import</button>
+            </form>
+        </div>
+    }
+    else
+    {
+        <nhs-error-summary title-text="This notification cannot be imported" title-id="error-summary-title"
+                           role="alert" tabindex="-1" data-module="error-summary">
```

Comment:
Hmmm I'm a little weary of the the tabindex -1.. Looking at this (https://webaim.org/techniques/keyboard/tabindex) 0 might be more appropriate.. what's the behaviour you're trying to achieve here?